src/scripts/lib/db.js

```diff
@@ -205,6 +234,10 @@ export default class Db {
       });
   }
 
+  setLocal (key, value) {
```

Comment:
Since we're doing `setLocal`, maybe we should also do `getLocal` to replace `localstorage.getItem()` calls